__tests__/app.test.js

```diff
@@ -165,4 +164,121 @@ describe('200: GET', () => {
         });
     });
 });
+describe('201: POST Request body accepts an object with the following properties, body, username', () => {
+    it('should return a 201 and an should post the comment object to the comment table in database', () => {
+        const requestBody = {
+            body: 'This is one of the best board games ever!',
+            username: 'bainesface',
+        };
+        return request(app)
+        .post('/api/reviews/1/comments')
+        .send(requestBody)
+        .expect(201)
+        .then(({ body }) => {
+            expect(body).toMatchObject({
+                author: expect.any(String),
+                body: expect.any(String),
+                comment_id: expect.any(Number),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+            });
+        });
+    });
+    it('should respond with a 400 if there is no comment in body value in the posted object', () => {
+        const requestBody = {
+            body: '',
+            username: 'bainesface',
+        };
+        return request(app)
+        .post('/api/reviews/1/comments')
+        .send(requestBody)
+        .expect(400)
+        .then(({ body }) => {
+            const { msg } = body;
+            expect(msg).toBe('No comment given')
+        });
+    });
+    it('should respond with a 400 if there is no body key in the posted object', () => {
+        const requestBody = {
+            username: 'bainesface',
+        };
+        return request(app)
+        .post('/api/reviews/1/comments')
+        .send(requestBody)
+        .expect(400)
+        .then(({ body }) => {
+            const { msg } = body;
+            expect(msg).toBe('No comment given')
+        });
+    });
+    it('should respond with a 400 if there is no value in the username in the posted object', () => {
```

Comment:
ditto to my above comment

Reply:
This now as above I have changed the wording to be more accurate :)